mate-panel/panel-applet-frame.c

```diff
@@ -827,6 +827,10 @@ _mate_panel_applet_frame_applet_broken (MatePanelAppletFrame *frame)
 	gtk_window_set_title (GTK_WINDOW (dialog), _("Error"));
 
 	gtk_widget_show (dialog);
```

Comment:
I think that `gtk_widget_show ()` is not needed if you use gtk_window_present...

Reply:
@muktupavels no, you are wrong

I have tested without gtk_widget_show and the dialog don't get the focus

Reply:
Hmm, I didn't think about it when fixing similar issue in https://github.com/mate-desktop/mate-polkit/commit/ed37b52fbfbce5d57d0549fac1a9dca2502f3372 and https://github.com/mate-desktop/mate-session-manager/commit/9e8d8ad24ad1025634b2558bc572937e55c64a2c. Didn't test without `gtk_widget_show` too. I guess it does no harm.